### PR TITLE
[dev] Fix comment

### PR DESCRIPTION
**Pull request by speedandfunction-anribolon**
Fix duplicate phrase in a comment.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -21394,7 +21394,8 @@
           "version": "2.1.1",
           "resolved": "https://registry.npmjs.org/is-extglob/-/is-extglob-2.1.1.tgz",
           "integrity": "sha1-qIwCU1eR8C7TfHahueqXc8gz+MI=",
-          "dev": true
+          "dev": true,
+          "optional": true
```
Comment (berndsi) [inline package-lock.json:21398]: Which npm version do you use? https://stackoverflow.com/a/55373854/1325669

Comment (niedzielski) [reply package-lock.json:21398]: @speedandfunction-anribolon, can you drop this file from the PR?

Comment (speedandfunction-anribolon) [reply package-lock.json:21398]: sure

Comment (niedzielski) [reply package-lock.json:21398]: I've cherry-picked [your patch here](https://gerrit.wikimedia.org/r/c/wvui/+/624877) and dropped this file.

**Pull request closed without merging**